src/compositemark/boxplot.ts

```diff
@@ -102,6 +103,8 @@ export function normalizeBoxPlot(
     config
   );
 
+  const invalid = markDef.invalid || null; //|| null to respect the existed default behavior
```

Comment:
Let's remove `|| null; //|| null to respect the existed default behavior` 
since the doc has been saying that filter is the default (for all marks). 

So the existing behavior to include `null` by default was a bug.